pelican/tests/test_settings.py

```diff
@@ -319,3 +320,11 @@ def test_coerce_overrides(self):
             'THEME_STATIC_DIR': 'theme',
         }
         self.assertDictEqual(overrides, expected)
+
+    def test_stringify_plugins(self):
+        settings = {
+            'PLUGINS': ['string_plugin', NoopPlugin]
+        }
+        stringify_plugins(settings)
+        # Plugins are converted to string references
+        self.assertTrue(all(str(p) == p for p in settings['PLUGINS']))
```

Comment:
It would also be nice to have an expected result for the value and test for that. Something like:
```python
self.assertCountEqual(
    ['string_plugin', 'NoopPlugin'],
    settings['PLUGINS']
)
```

PS: Of course the value `'NoopPlugin'` would depend on how you stringify them.

Reply:
Yeah this is a remnant of not really having a good way of turning the module into a string. Using your suggestion from above this should work just fine.